doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -557,14 +557,18 @@ This creates a derivation with a directory structure like the following:
 
 ## `writeReferencesToFile` {#trivial-builder-writeReferencesToFile}
 
-Writes the closure of transitive dependencies to a file.
+Deprecated. Use [`writeReferenceClosureToFile`](#trivial-builder-writeReferenceClosureToFile) instead.
+
+## `writeReferenceClosureToFile` {#trivial-builder-writeReferenceClosureToFile}
+
+Writes the closure of transitive dependencies of the given paths to a file.
```

Comment:
Nit: to me this almost reads like we're given "paths" which are "paths to a file", rather than "we write to a file".

Just so I propose at least something: "given a list of store paths (???), writes the closure of their transitive dependencies to a file".

Reply:
It's directly taken from the documentation of `writeReferencesToFile`, with "the given path" -> "the given paths".

The new version sounds more clearly.

Reply:
I modify the doc. Please take a look.